client/jetpack-cloud/sections/agency-dashboard/sidebar/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { useTranslate } from 'i18n-calypso';
+import { useDispatch } from 'react-redux';
+import JetpackIcons from 'calypso/components/jetpack/sidebar/menu-items/jetpack-icons';
+import SiteSelector from 'calypso/components/site-selector';
+import Sidebar from 'calypso/layout/sidebar';
+import SidebarFooter from 'calypso/layout/sidebar/footer';
+import SidebarItem from 'calypso/layout/sidebar/item';
+import SidebarMenu from 'calypso/layout/sidebar/menu';
+import SidebarRegion from 'calypso/layout/sidebar/region';
+import CurrentSite from 'calypso/my-sites/current-site';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import 'calypso/components/jetpack/sidebar/style.scss';
+import type { CalypsoDispatch } from 'calypso/state/types';
+import type { FunctionComponent } from 'react';
+
+interface Props {
+	path: string;
+}
+const DashboardSidebar: FunctionComponent< Props > = ( { path } ) => {
+	const translate = useTranslate();
+	const reduxDispatch = useDispatch< CalypsoDispatch >();
+	const onNavigate = ( menuItem: string ) => () => {
+		reduxDispatch(
+			recordTracksEvent( 'calypso_jetpack_sidebar_menu_click', {
+				menu_item: menuItem,
+			} )
+		);
+
+		window.scrollTo( 0, 0 );
+	};
+
+	const onGetHelp = () => {
+		reduxDispatch(
+			recordTracksEvent( 'calypso_jetpack_sidebar_menu_click', {
+				menu_item: 'Jetpack Cloud / Support',
+			} )
+		);
+		window.scrollTo( 0, 0 );
+	};
```

Comment:
I think we can use `onNavigate` instead of this function since they both track the same event and do the same behavior.